lib/vsc/utils/script_tools.py

```diff
@@ -102,7 +103,25 @@ def _merge_options(options):
     return opts
 
 
-class ExtendedSimpleOption(SimpleOption):
+class ConfigOption:
+    """
+    Allow using configargparse.ArgParser instead of GeneralOption but with the same
```

Comment:
i'm quite sure that `same` doesn't mean what you think it means ;)
anyway, best to start with a separate named class, and start to swap it in controlled way. then we can see what features are not used.

but tbh, i fail to see the need at all. if you want to make a general option with this configargparse as backend, then go for that and do this in vsc-base.

Reply:
I want to get rid of GeneralOption altogether.

Reply:
not sure why you want this, but starting in this repo is not the correct place imho.
it's better to have a look what "features" of generaloption we don't want anymore, and then look for code where they are used; and then estimate the amount of work to replace it.